skfda/preprocessing/dim_reduction/feature_extraction/__init__.py

```diff
@@ -1,5 +1,9 @@
-"""Feature extraction."""
-from ._ddg_transformer import DDGTransformer
-from ._fda_feature_union import FDAFeatureUnion
-from ._fpca import FPCA
-from ._per_class_transformer import PerClassTransformer
+import warnings
+
+from ... import feature_construction
+
+warnings.warn(
+    'The module "feature_extraction" is deprecated.'
+    'Please use "feature_construction"',
```

Comment:
I think that existing use cases involve more frequently the `FPCA` class, so redirect them accordingly.